### PR TITLE
fix(deis-logger/manifests/deis-logger-svc.yaml): make the logger service run on port 80

### LINKED ISSUE #136: Make the deis-dev/logger service serve http on port 80
Ref deis/deis#4853
Ref deis/deis#4809

### PR DESCRIPTION
**Pull request by arschles**
Fixes #136
Ref deis/deis#4853

cc/ @jchauncey

### CODE REVIEW

**Comment by jchauncey** [comment]
@krancour

**Comment by arschles** [comment]
@jchauncey or @helgi were either of you able to test this by chance?

**Comment by jchauncey** [comment]
no but i will really quick

**Comment by jchauncey** [comment]
so i thought I had the proper env var for the service port in the controller but i was wrong. I submitted a PR to fix that before this one can be merged.

**Comment by arschles** [comment]
@jchauncey the PR is https://github.com/deis/workflow/pull/476, right?

**Comment by jchauncey** [comment]
yes

File: deis-logger/manifests/deis-logger-svc.yaml [hunk 1]
```diff
@@ -8,7 +8,8 @@ metadata:
     app: deis-logger
 spec:
   ports:
-  - port: 8088
+  - port: 80
+    targetPort: 8088
     name: http
     targetPort: http
```
Comment (arschles) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: I missed this and I think I need to remove it. @jchauncey objections?

Comment (jchauncey) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: remove what the targetPort?

Comment (arschles) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: on second thought, I think it should be sufficient to leave this as-is and remove my `targetPort` addition. The result can then be

``` yaml
- port: 80
  targetPort: http
```

thoughts?

Comment (jchauncey) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: This is how my manifest looks that I tested with - 

```
  - port: 80
    name: http
    targetPort: http
```

Comment (arschles) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: k, changing

Comment (arschles) [inline deis-logger/manifests/deis-logger-svc.yaml:13]: https://github.com/arschles/deis-charts/commit/fefd7970c3a24ff25cb30bde6657d09cbc76cc51

**Comment by jchauncey** [comment]
@arschles ok the other PR has been merged

**Comment by arschles** [comment]
@jchauncey thanks. I'm gonna wait til the image is built to merge this

**Pull request merged**